packages/pigeon/platform_tests/ios_unit_tests/ios/RunnerTests/AsyncHandlersTest.m

```diff
@@ -76,6 +77,10 @@ - (void)calculate:(Value* _Nullable)input
   }
 }
 
+- (void)voidVoid:(nonnull void (^)(FlutterError* _Nullable))completion {
```

Comment:
This name is... unusual. Is this following some defined convention?

Reply:
Yea, if you check out the pigeon file it means a function that takes no parameters and has no return type `void foo(void)`.